crates/miden-objects/src/note/note_tag.rs

```diff
@@ -440,7 +494,7 @@ mod tests {
         let private_account_id = AccountId::try_from(PRIVATE_ACCOUNT_INT).unwrap();
 
         // Expected private tag with LOCAL_EXECUTION_WITH_ALL_NOTE_TYPES_ALLOWED.
```

Comment:
Let's update the `LOCAL_EXECUTION_WITH_ALL_NOTE_TYPES_ALLOWED` with the new `NoteTag::xyz` variant.